src/adapter/src/coord/command_handler.rs

```diff
@@ -285,6 +285,34 @@ impl Coordinator {
                         ));
                     let _ = tx.send(oracle);
                 }
+
+                Command::DetermineRealTimeRecentTimestamp {
+                    source_ids,
+                    real_time_recency_timeout,
+                    tx,
+                } => {
+                    let result = self
+                        .determine_real_time_recent_timestamp(
+                            source_ids.iter().copied(),
+                            real_time_recency_timeout,
+                        )
+                        .await;
+
+                    match result {
+                        Ok(Some(fut)) => {
+                            task::spawn(|| "determine real time recent timestamp", async move {
```

Comment:
This corresponds to the spawning in the old peek sequencing's `peek_real_time_recency`.